modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ObjectField/useObjectFieldForm.d.ts

```diff
@@ -31,6 +31,8 @@ export declare function useObjectFieldForm({
 	errors: import('@liferay/object-js-components-web').FormError<
 		ObjectField & {
 			function: unknown;
+			script: unknown;
```

Comment:
@manuelecastro There's a SF failure and it seems to be related with this file you've commited. I think you can just remove it because it's not related with your changes.